File: services/telegramservice/DTOs/DTO/MessageDTO.cs
using System;

namespace DTOs.DTO;

public class MessageDTO
{
    public long? IdGroup { get; set; }
    public long? IsComments { get; set; }
    public long? ReplyToPost { get; set; }
    public long? IdMessage { get; set; }
    public long? IdUser { get; set; }
    public long? IdGroupedMessage { get; set; }
    public string? ContentText { get; set; }
    public string? ContentMedia { get; set; }
    public long? IdReply { get; set; }
    public DateTime? Date { get; set; }
}

File: services/dbservice/src/main/java/com/torin/dbService/dto/MessageDto.java
package com.torin.dbService.dto;

import java.time.Instant;

public record MessageDto(
        Long idGroup,
        Long isComments,
        Long replyToPost,
        Long idMessage,
        Long idUser,
        Long idGroupedMessage,
        String contentText,
        String contentMedia,
        Long idReply,
        Instant date) {
}
